### PR TITLE
feat: added badge on avatar

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added badge on the avatar. It contains:

>Online Badge
Offline Badge

>Small
Medium
Large

badgeOnAvatar

### CODE REVIEW

File: components/badge-on-avatar.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.online-badge {
+    display: inline-block;
+    position: relative;
+}
+
+.green-dot {
```
Comment (FarhanMobashir) [inline components/badge-on-avatar.css:6]: make a dot class with all the basic styling and then make a green dot class for green color and for red repectively

**Review by FarhanMobashir — CHANGES_REQUESTED** [review]
code looks good.

**Review by FarhanMobashir — APPROVED** [review]
Code looks neat. ✔️

**Review by FarhanMobashir — APPROVED** [review]

**Review by saadahmadhere — APPROVED** [review]
Code looks neat, nice use of variables. Good to merge.

**Pull request merged**